examples/multicached_decorator.py

```diff
@@ -39,6 +49,8 @@ async def test_multi_cached():
     await cache.delete("d")
     await cache.close()
 
+    await multi_cached_key_builders(keys=(1, 2)) == {1: "Z", 2: "Y"}
```

Comment:
I'm not sure we need more in the example, so I'd suggest reverting the changes to this file (the code is wrong currently anyway).